jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/multi/ProcessInstanceDetailsMultiPresenter.java

```diff
@@ -248,17 +248,17 @@ public void callback(List<NodeInstanceSummary> completedNodes) {
                                                             .addParameter("completedNodes", completedNodeParam.toString())
                                                             .addParameter("readOnly", "true")
                                                             .addParameter("processId", processId)
-                                                            .addParameter("deploymentId", selectedDeploymentId)));
+                                                            .addParameter("processInstanceId", selectedProcessInstanceId)));
```

Comment:
Seems like this is breaking the process instance diagram view. I think these changes should be done but perhaps in a separated PR as they are somehow unrelated to the main issue.